homeassistant/helpers/template.py

```diff
@@ -1854,8 +1854,6 @@ def relative_time(value):
         return value
     if not value.tzinfo:
         value = dt_util.as_local(value)
-    if dt_util.now() < value:
```

Comment:
The comment for this function specifically says:
>  Make sure date is not in the future, or else it will return None.

But it doesn't look like that's actually the case with the existing code.  And I'm not sure what the new code does either.  Can we verify the behavior and update the comment to make it clear what the expected result is with a datetime in the future?